### ISSUE TITLE
[Accessibility] Unable to insert the role or Update the tab index dynamically

### ISSUE THREAD

**Issue by AkashSurana** [kekeh/mydatepicker#638]
Hello Team,

Issue:
We are getting 49 errors when we run the ARCTOOL KIT
![image](https://user-images.githubusercontent.com/27774371/88265247-c2c00880-ccea-11ea-93e0-eab47ae270aa.png)

I want to update either the role for each of the "td" in the calendar or need to update the tab index to "-1"

The dynamic code which I am using:
ngAfterViewInit(){
        var allTDElements = document.getElementsByClassName("caltable")[0].getElementsByTagName("td")

        for (var i = 0; i < allTDElements.length - 1; i++) {
            allTDElements[i].attributes['tabindex']['value'] = "-1";
            allTDElements[i].setAttribute("role", "button");
        }
    }

The code is running fine but the values are not reflecting in the component. Can you please help me on how can I update either the role or tab index value

**Comment by kekeh** [comment]
Hi @AkashSurana 

I am not sure what you are trying but, implement [this](https://github.com/kekeh/mydatepicker#calendarviewchanged-callback) or [this](https://github.com/kekeh/mydatepicker#calendartoggle-callback) callback and put your loop inside the callback function.

If it doesn't succeed try same with Angular ngZone.
